src/commonmark/en/content/developer/web-api.md

```diff
@@ -12260,7 +12260,7 @@ event data. A request looks like this:
       &dimension=ou:LEVEL-2&clusterSize=100000
       &bbox=-13.2682125,7.3721619,-10.4261178,9.904012&includeClusterPoints=false
 
-The cluster response provides the count of underlying points, the centre
+The cluster response provides the count of underlying points, the center
```

Comment:
the documentation contains both UK and US variations of English, so either of these are OK.